cfgov/unprocessed/apps/teachers-digital-platform/css/pages/survey-results.less

```diff
@@ -1,3 +1,14 @@
+@media not print {
```

Comment:
Just a thought but should you maybe make it `display: none` by default and display it with `@media print`? I haven't looked into the support for `@media` versus `@media not`, but negating the hiding seems safer for any browser that may not support negated media queries.

Reply:
Updated.